opennms-config/src/main/java/org/opennms/netmgt/config/UserManager.java

```diff
@@ -127,8 +124,9 @@ public void parseXML(final InputStream in) {
             }
         
             _buildDutySchedules(m_users);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
```

Comment:
Sonar doesn't complain about throwing runtime exception ?

Reply:
thanks for noticing, fixed now.